src/manifolds/FixedRankMatrices.jl

```diff
@@ -261,9 +261,6 @@ function project!(::FixedRankMatrices, Y::UMVTVector, p::SVDMPoint, A::AbstractM
     Y.Vt .= (aTu - p.Vt' * uTav')'
     return Y
 end
-function project!(M::FixedRankMatrices, Y::UMVTVector, p::SVDMPoint, X::UMVTVector)
-    return project!(M, Y, p, X.U * X.M * X.Vt)
```

Comment:
We have to adapt the tests, since this was deleted (and sure this method was wrong, one would need the point `q` to whose tangent space the vector `X` belongs to, which we do not have). We could introduce a vector transport by projection using something like this.